app/views/layouts/application.html.erb

```diff
@@ -10,9 +10,9 @@
   <h1>わびさび Wabi-sabi</h1>
 
   <% flash.each do |type, msg| %>
-    <%= content_tag :div, msg, class: type %>
+    <%= content_tag :div, sanitize(msg), class: type %>
```

Comment:
We were getting extra characters put into the msg output for building the flash response. I found this awesome rails method that strips those unwanted characters off so it can read it as valid HTML. The only characters it doesn't protect against are( <  > &).